target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -159,6 +159,9 @@ ramips_setup_interfaces()
 		ucidef_set_interface_lan "lan1 lan2 lan3 lan4"
 		ucidef_set_interface "qtn" ifname "eth1" protocol "static" ipaddr "1.1.1.1" netmask "255.255.255.0"
 		;;
+	ruijie,rg-ew1200g-pro)
+		ucidef_set_interfaces_lan_wan "lan3 lan2 lan1" "wan"
```

Comment:
merge it to case `ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"`

Reply:
But ports on the router are labeled as "WAN LAN3 LAN2 LAN1" (if reading from left to right)

Reply:
If the device has Ethernet port number labels, there is no need to do so. At least I didn't see any devices sorting port sequences in OpenWrt.